common/src/main/java/io/netty/util/internal/StringUtil.java

```diff
@@ -388,6 +388,68 @@ public static CharSequence escapeCsv(CharSequence value) {
     }
 
     /**
+     * Unescapes the specified escaped CSV field, if necessary according to
+     * <a href="https://tools.ietf.org/html/rfc4180#section-2">RFC-4180</a>.
+     *
+     * @param value The escaped CSV field which will be unescaped according to
+     *              <a href="https://tools.ietf.org/html/rfc4180#section-2">RFC-4180</a>
+     * @return {@link CharSequence} the unescaped value if necessary, or the value unchanged
+     */
+    public static CharSequence unescapeCsv(CharSequence value) {
+        int length = checkNotNull(value, "value").length();
```

Comment:
Consider removing explicit null check ... You "dereferense" on this line anyways and NPE will already be thrown. There is also only 1 argument so I don't think much context will be lost by not having "value" in the exception message.